nextflow.config

```diff
@@ -14,15 +14,18 @@ params {
   name = false
 
   // Initializing parameters
-  cprimers = false
-  vprimers = false
   set_cluster_threshold = false
-  cluster_threshold = false
+  cluster_threshold = -1
```

Comment:
Maybe use a more reasonable default value for this, in case sb. runs the pipeline without setting it?

Reply:
0.14?